lib/resources/r4/charge_item.yaml

```diff
@@ -105,3 +105,16 @@ fields:
         - display: Bill Code Schedule
           system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14002
           info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-14002-bill-code-schedule
+
+  - name: National Drug Product Extension
+    terminology_name: extension[x].valueCodeableConcept
```

Comment:
Should these nested bindings be extension.extension[x].valueCodeableConcept? cc @exopsRamba

Reply:
Updated in multiple places here - https://github.com/cerner/fhir.cerner.com/pull/683/commits/7e3a1cf41a4c535e569612799fcfd545d2f382fa

Reply:
Updated screenshots

Reply:
+1. Thanks for making that change.

Reply:
The revenue-code binding can stay the way it was since the CodeableConcept for it isn't a nested extension.

Reply:
added one more terminology binding update here - https://github.com/cerner/fhir.cerner.com/pull/683/commits/71a07816848b6cc6b813b814486ea8442ca69e48

reverted revenue-code update here - https://github.com/cerner/fhir.cerner.com/pull/683/commits/f02628d9b960b288ccdab094289064d0765c5695